dnslol/dnslol.go

```diff
@@ -205,18 +253,16 @@ func (e Experiment) buildQueries(name string) []query {
 // other than RcodeSuccess, a false bool and a nil error or an empty rcode
 // string, a false bool and a non-nil error. In all cases the queryTimes latency
 // stat is updated for the server and query type performed.
-//
-// TODO(@cpu): Update comment
```

Comment:
I think this comment still needs updating.

Reply:
Oops, good catch. Thanks!

Reply:
Updated in 526186a